### PR TITLE
Add Release Notes for LangSmith

### PR DESCRIPTION
**Pull request by barberscott**
Draft PR for first pass at adding release notes.

### CODE REVIEW

**Comment by hinthornw** [comment]
Do we want to add a workflow to populate this automatically? or keep manual for now?

**Comment by barberscott** [comment]
> Do we want to add a workflow to populate this automatically? or keep manual for now?

@hinthornw I expect we'll want to get there eventually.  For now, manual.

File: docs/release_notes/20240129.mdx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- None
+
+## Notes for Self-Hosted Deployments
+
+### Infrastructure Changes
+
+- Added the `clickhouse` database service.  Run results will now be stored in ClickHouse instead of Postgres to improve performance and scalability and reduce delays in the time it takes for runs to appear in LangSmith.
+    - Note that if you wish to retain access to run data in the Langsmith UI after updating, a data migration will need to be performed.  Details are available at https://github.com/langchain-ai/helm/blob/main/charts/langsmith/docs/UPGRADE-0.2.x.md
+
+### Admin changes
```
Comment (langchain-infra) [inline docs/release_notes/20240129.mdx:59]: the duplicate of admin changes header is a bit confusing but cant think of a better way haha, maybe call it Self-Hosted Admin changes?

**Review by langchain-infra — APPROVED** [review]
overall lgtm

**Pull request merged**